### ISSUE TITLE
Strange background appears randomly

### ISSUE THREAD

**Issue by binaryplease** [enkore/i3pystatus#192]
On Arch Linux (up to date) this occures from time to time:

![2015-03-30-000021_1919x33_scrot](https://cloud.githubusercontent.com/assets/1719781/6888246/2b6efcfc-d678-11e4-85a5-6f4c9e88b9e0.png)

The size of the colored field varies and seems to start growing from the right to the left. It is almost always there after wake up and after the boot.

Restarting i3 with ctrl+shift+r gets rid of it.

Is it only me?

My config:

````python

# -*- coding: utf-8 -*-

import subprocess

from i3pystatus import Status
from i3pystatus.mail import imap

status = Status(standalone=True)


#Date
status.register("clock",
        format="%a %-d %b %H:%M:%S",)

#CPU Temperature
status.register("temp",format="{temp}°C",file="/sys/class/thermal/thermal_zone1/temp")

status.register("cpu_usage",format="{usage_cpu0:3.0f}%{usage_cpu1:3.0f}%{usage_cpu2:3.0f}%{usage_cpu3:3.0f}%{usage_cpu4:3.0f}%{usage_cpu5:3.0f}%{usage_cpu6:3.0f}%{usage_cpu7:3.0f}%")

status.register("text", text="CPU",color="#00ffff")
#status.register("text", text="CPU",color="#00ffff",cmd_leftclick="urxvt -e bash -c 'htop && zsh'")

#RAM
status.register("mem",format="{percent_used_mem}%",)
#status.register("mem",format="{percent_used_mem}% ({used_mem}/{total_mem})",)

status.register("text", text="RAM",color="#00ffff")
#status.register("text", text="RAM",color="#00ffff",cmd_leftclick="urxvt -e bash -c 'htop && zsh'")

#Battery
status.register("battery",
    battery_ident="BAT1",
    format="{status}/{consumption:.2f}W {percentage:.2f}% {remaining:%E%hh:%Mm}",
    alert=True,
    alert_percentage=5,
    status={
        "DIS": "↓",
        "CHR": "↑",
        "FULL": "=",
    },)

status.register("text", text="BATTERY",color="#00ffff")
#status.register("text", text="BATTERY",color="#00ffff",cmd_leftclick="urxvt -e bash -c 'acpi && zsh'")

# Network 
status.register("runwatch",
    name="DHCP",
    path="/var/run/dhc*.pid",)

status.register("network",
    interface="enp3s0",
    format_up="{v4} {bytes_recv:04.0f}kbs",)


status.register("network",
    interface="wlp4s0",
    format_up="{essid} {quality:03.0f}% {v4} {bytes_recv:04.0f}kbs",)

#status.register("network_traffic",
##    interface="wlp3s0",
#    format="{bytes_recv}/{bytes_sent}")

status.register("text", text="NETWORK",color="#00ffff")
#status.register("text", text="NETWORK",color="#00ffff",cmd_leftclick="urxvt -e bash -c 'ip addr && zsh'")

#Disk
status.register("disk",
    path="/",
    format="/ {percentage_used}% [{avail}G]",)
#status.register("text", text="DISK",color="#00ffff",cmd_leftclick="urxvt -e bash -c 'lsblk && df && zsh'")
status.register("text", text="DISK",color="#00ffff")

#Audio
status.register("alsa",
    format="♪ {volume}",format_muted="♪ MUTE")
#status.register("text", text="VOLUME",color="#00ffff",cmd_leftclick="urxvt -e bash -c 'alsamixer && zsh'",cmd_rightclick="bash -c 'amixer set Master toggle $> /dev/null'")
status.register("text", text="VOLUME",color="#00ffff")


#Backlight
status.register("backlight",
    format="☀ {percentage}",)

#Mail
status.register("mail",
    backends=[
        imap.IMAP(
             # port and ssl are the defaults
             host="imap.googlemail.com",
             username="binaryplease@gmail.com", password="password"
            )
    ],hide_if_null=False,format="G: {unread}")

status.register("mail",
    backends=[
        imap.IMAP(
             # port and ssl are the defaults
             host="imap.1und1.de",
             username="pablo.ovelleiro@committance.com", password="password"
            )
    ],hide_if_null=False,format="C: {unread}")

status.register("text", text="MAIL",color="#00ffff")
#status.register("text", text="MAIL",color="#00ffff",cmd_leftclick="thunderbird &> /dev/null")

#Weather
status.register("weather",
    format="{current_temp} - {humidity}%",location_code="GMXX5686",colorize=True)

status.run()



````

**Comment by facetoe** [comment]
I get this sometimes also. It usualy happens when a module errors out and prints a long error message. I can see in the left of the screen you have an error so that's probably what is causing this. 

I've submitted a patch (#190) to truncate these long error messages on a module by module basis, but it might be worth applying it to all modules and printing the full stacktrace to a log instead as it is really annoying when this happens.

**Comment by enkore** [comment]
Known bug with i3bar if the text length exceeds the bar width

**Comment by Arvedui** [comment]
This should be fixed with 3.10.1 :)

From the changelog:
> i3bar: cut long statuslines from the left
